log/go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/cosmos/cosmos-sdk/log
 
 go 1.19
+
+require github.com/tendermint/tendermint v0.37.0-rc2
```

Comment:
🥲

Reply:
yea not happy about this either. Not sure what the alternative is other than wrapping our logger with a different interface with one that can be passed to tendermint. This way we define our own interface. I know @alexanderbez has some thoughts here. This may be the cleanest, but would be somewhat of a hefty breaking change.

Reply:
Looks like this was resolved? 

Maybe we can convince the Comet team to modify their logging interface so we can use structured typed fields.

They could even have their logger be a generic: `type Logger [TypedLogger | UntypedLogger]` where `UntypedLogger` is the logger interface that exists today and `TypedLogger` would be a new one that supports typed fields.

Reply:
cc @thanethomson

Reply:
Merging this for now as I think there is a way in the future to be able to define our own interface and only pass the interface here to tendermint in start. There may be no need for changes on the tendermint side